WORKSPACE

```diff
@@ -45,10 +53,12 @@ py_repositories()
 
 ############################################
 # Aspect bazel-lib
-load("@aspect_bazel_lib//lib:repositories.bzl", "register_coreutils_toolchains")
+load("@aspect_bazel_lib//lib:repositories.bzl", "register_coreutils_toolchains", "register_tar_toolchains")
 
 register_coreutils_toolchains()
 
+register_tar_toolchains()
```

Comment:
Users will also need this toolchain? Is the expectation that they register too, or should this ruleset be doing that in the setup